frontend/yarn.lock

```diff
@@ -10770,3 +10770,4 @@ yargs@^11.0.0:
     which-module "^2.0.0"
     y18n "^3.2.1"
     yargs-parser "^9.0.2"
+    
```

Comment:
This file is auto generated and shouldn't get changed.

Reply:
Yeah, that is because I removed all me previous changes related to these new packages that I installed for testing.
So I put back the latest version before these tests.
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/279/commits/d76f87c11a34fbc61dde7a710a0671d9e253b00b